numpy/distutils/system_info.py

```diff
@@ -414,6 +414,89 @@ def get_standard_file(fname):
     return filenames
 
 
+def _parse_order(base_order, order_str):
+    """ Parse the list of `order_str` by splitting with "," and only returning elements from `base_order`
+
+    This method will sequence `order_str` and check for their invidual elements in `base_order`.
+
+    The items in `order_str` may be negated via '^item' or '!itema,itemb'.
+    The `order_str` must start with ^ to negate all options.
+
+    Raises
+    ------
+    ValueError: for mixed negated and non-negated orders or multiple negated orders
+
+    Parameters
+    ----------
+    base_order : list of str
+       the base list of orders
+    order_str : str or None
+       the orders to be parsed, if none, `base_order` is returned
+
+    Returns
+    -------
+    allow_order : list of str
+        allowed orders in lower-case
+    reject_order : list of str
+        rejected orders in lower-case
```

Comment:
This return value seems pointless, since none of the callers use it

Reply:
Yeah, I didn't know whether this could be useful for info in some build-processes.

I am happy to delete it, if you want?

Reply:
deleted